app/views/notes/_comment.html.erb

```diff
@@ -85,6 +85,7 @@
                 <img alt="React to post" class="emoji" height="20" width="20" src="<%= emoji_image_map[e] %>">
               <% end %>
               <% end %>
+            </div>
```

Comment:
I think this is a close `<div>` in the wrong place, maybe... causing these errors:

```
        expected to find css "#c474456746 .comment .comment-body p" but there were no matches
Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find css ".alert-success"
```